.github/workflows/python.yml

```diff
@@ -47,41 +47,41 @@ jobs:
     name: ${{ matrix.title }}
     runs-on: ubuntu-latest
     if: ${{ !contains(github.event.pull_request.title, 'WIP') }}
-    timeout-minutes: 60
+    timeout-minutes: 90
```

Comment:
I've done this to fix the MacOS 12 job that was timing out after 60 minutes. Happy to revert if we think we should tackle the issue differently or on another PR.

Reply:
It seems that we use bundled Protobuf instead of using Homebrew's Protobuf:
https://github.com/apache/arrow/actions/runs/5267649495/jobs/9523177390#step:5:236

```text
-- Could NOT find Protobuf (missing: Protobuf_LIBRARIES Protobuf_INCLUDE_DIR) (Required is at least version "3.0.0")
```

I should have confirmed this from CI log explicitly in #36029...

Reply:
Would you revert it back to 60 minutes and fix that on a different PR? I think that's probably better then.